gallery/templates/view_dir.html

```diff
@@ -32,7 +32,38 @@
             </ul>
         </div>
     </div>
-{% if children|length >= 1 %}
+{% if children|length + pin_children|length >= 1 %}
+    <div class="row" id="pin_child_list">
+    {% for child_type, child in pin_children %}
+        {% if not child.hidden or (auth_dict['is_eboard'] or auth_dict['is_rtp'])  or auth_dict['is_alumni'] %}
+            <div class="col-md-3 col-sm-4 col-xs-6">
+                <div class="panel panel-warning">
```

Comment:
Would it be cleaner to do sorting in python and do the bg selection based on the pinned attribute of an item when rendering?

Reply:
Hmmmm I will look into this